### DISCUSSION TITLE
Can’t wait

### DISCUSSION THREAD

**Discussion by hhv** [ToonVanEyck/OpenFlap#8] [General]
Hi, I am glad to see you are still working on this project Toon. I'm looking forward to seeing the "work in progress" disappear from the Readme and contributing to the project.

Regards,

Huib.

**Comment by shagadeli** [comment]
I am too periodically checking this GitHub to start building mine. Is it safe to start ordering the hardware Toon?

**Reply by ToonVanEyck** [reply]
I redid most of the hardware in KiCad, coming from fusion 360. But I have not been able to order and validate those designs yet. So unless you really know what you are doing, I would still wait a bit. 

I am currently working on improving the controller software as it is not yet a 100% reliable and doesn't really provide a lot of feedback to the user when things do go wrong. 

I'm curious, if provided with the option, would you prefer purchasing this from me in diy-kit form? How many modules would you want to build?

**Reply by hhv** [reply]
Hi Toon, 

while I am curious and would like to, and will, order and build a few of this modules myself, I would definitely appreciate it if I could buy these modules from you and start working on my project. I need about 28 of these modules. So, if these modules come pre-built that would be best, but a diy-kit is also OK for me. 

Regards,
Huib.

**Reply by ToonVanEyck** [reply]
Hi Huib, 

I haven't even tested with 28 modules myself, So I would definitely wait a couple more months. Once I finish reworking the controller hardware & firmware, I will build a large display to validate the project as a whole. I would hate for you to spend time a money on something that doesn't work properly.

**Reply by hhv** [reply]
Hi Toon, I can see you're still working on Open Flap, and the message on the homepage is still "wait a bit". 

Wait. I don't want to sound pushy. Is there anything I can do to help? I don't mind spending some time and money even if the outcome is uncertain. 

By the way, did I see some Bluetooth LE code in the commits?

If everything is still changing a lot I don't mind waiting, but if I can build and test stuff let me know.

Kind regards,

Huib.

**Reply by ToonVanEyck** [reply]
Hi Huib,

I'm currently on a bit of a break regarding the project but I will definitely resume it in a couple of months. 

I was working on a test setup for the hardware to do automated testing when I started my brake.

The hardware should be quite stable, I have 5 modules (not a lot) which are working excellent. Once I have a test setup I will order more so I can automatically test them. 

I feel confident that you could do something with the hardware as currently available in this repo. However I cannot commit a lot of time in supporting anyone at the current time (because I'm on a break from the project) and at the current state of the project (I don't want to support individual people and would rather spend that time on writing a manual that individual people can use).

So before you dive in, make sure you are comfortable with flashing arm chips, esp's, running devcontainers, etc...
